requirements/extra.txt

```diff
@@ -6,5 +6,6 @@ torchtext>=0.9.*
 omegaconf>=2.0.5
 hydra-core>=1.0.5
 jsonargparse[signatures]>=4.6.0
+docstring-parser==0.13
```

Comment:
@carmocca Could you add a comment next to it on why we pinned it?

Reply:
There's git blame for that

Reply:
can we have some limited range? `docstring-parser<=0.13`

Reply:
```suggestion
docstring-parser<=0.13
```